tests/btrfs-progs/install.pm

```diff
@@ -24,22 +24,17 @@ sub log_create {
 }
 
 sub install_dependencies {
-    my @deps = qw(
-      autoconf
-      automake
-      lzo-devel
-      git-core
-      gcc
-      libblkid-devel
-      zlib-devel
-      libext2fs-devel
-      libmount-devel
-      libuuid-devel
-    );
-    if (get_var('BTRFS_PROGS_DEPS')) {
-        @deps = split(/,/, get_var('BTRFS_PROGS_DEPS'));
+    my $transaction = shift;
```

Comment:
Why not use `is_transactional` directly here in install_dependencies, instead of passing the parameter `$transaction` into the function?